README.md

```diff
@@ -57,4 +59,9 @@ Now you can run the project's `npm` scripts like usuual:
 
 **_Note_**: changes are bi-directional
 
-Learn more about Docker [here](https://docs.docker.com/get-started/) or [configuring Headless Chrome for your own machine](https://github.com/GoogleChrome/puppeteer/blob/master/docs/troubleshooting.md).
\ No newline at end of file
+Learn more about Docker [here](https://docs.docker.com/get-started/) or [configuring Headless Chrome for your own machine](https://github.com/GoogleChrome/puppeteer/blob/master/docs/troubleshooting.md).
+
+#### Workflows
```

Comment:
@anthonygore 
Wondering if you had any objections to renaming 
- the `build` task to `release`
- the `ci` task to `build`

Seems a little bit more semantic in the context of these newer workflows